cluster/kubemark/config-default.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```

Comment:
How redundant with gce/config-default.sh is this? Are we going to need to remember to update both of them as we add new parameters?